backend/services/classService.js

```diff
@@ -0,0 +1,71 @@
+// services/classService.js
+const Class = require('../models/Class');
+
+// Function to create a new class
```

Comment:
use git ignore , node modules should not appear, I can't review when node files are also commited